src/home/homework-luks.c

```diff
@@ -1866,6 +1867,7 @@ static int make_partition_table(
         _cleanup_free_ char *disk_uuid_as_string = NULL;
         uint64_t offset, size, first_lba, start, last_lba, end;
         sd_id128_t disk_uuid;
+        fdisk_sector_t fdisk_sector_size;
```

Comment:
that's such a weird type. I think it would be better to just use uint64_t here.

and maybe drop the "fdisk" prefix, we don't name ther other fields we retrieve from libfdisk with such a prefix either